cypress/commons/actions/generic/Login.js

```diff
@@ -3,81 +3,66 @@
 
 import 'cypress-localstorage-commands';
 import { GENERIC_SELECTORS } from '../../constants/generic/IdConstants';
-import { Scenarios } from './Scenarios';
-import { apiUtils as api } from '../../utils';
+import { apiUtils as api, authUtils as auth, routeUtils as route } from '../../utils';
 import { stub } from '../../services/stubbing';
-import { BREWERY_WORKSPACE_ID } from '../../constants/generic/TestConstants';
+import { setup } from '../../utils/setup';
+import { USER_EXAMPLE } from '../../../fixtures/stubbing/default/users';
 
 const BASE_URL = Cypress.config().baseUrl;
-function getMicrosoftLoginButton() {
-  return cy.get(GENERIC_SELECTORS.login.microsoftLoginButton);
-}
 
-function _handleWorkspaceSelector(url) {
-  if (url == null) {
-    url = BASE_URL;
-    const defaultWorkspaceId = stub.isEnabledFor('GET_WORKSPACES')
-      ? `/${stub.getDefaultWorkspaceId()}`
-      : `/${BREWERY_WORKSPACE_ID}`;
-    url += defaultWorkspaceId;
-  }
-  return url;
+function getDevLoginButton() {
```

Comment:
As previously discussed, it's not used by the current authentication mechanism, but it may be needed when we support other providers than MSAL, so we might as well keep it for now :slightly_smiling_face: